PAN-21: Slate Document Update: Updating the new property 'note' in EP… (#4)
evoraglobal/sieraapi-docs: source/includes/_epcs.md

      "fri": false,
      "epcExempt": false
    },
    "comment": "[2021-11-09] I'm a comment!",

>     "note" : "[2021-11-09] I'm a note!"

Reviewer: ```suggestion
    "note" : "A single EPC comment"
```

    },
    "scope": "WholeBuilding",
    "unit": null,
    "comment": "[2021-11-09] I'm a comment!",

>     "note" : "[2021-11-09] I'm a note!"

Reviewer: ```suggestion
    "note" : "A single EPC comment"
```

| `unit.fri`                   | **boolean**<br/>A boolean flag indicating if the current lease is [Full Repairing and Insuring (FRI)](https://www.herrington-carmichael.com/full-repairing-and-insuring-lease/) or not |
| `unit.epcExempt`             | **boolean**<br/>A boolean flag indicating if this unit has an [EPC exemption](https://www.gov.uk/energy-performance-certificate-commercial-property/exemptions) or not |
| `comment`                    | **string**<br/>The latest comment that has been recorded on the EPC record |

> | `note`                    | **string**<br/>The latest note that has been recorded on the EPC record |

Reviewer: ```suggestion
| `note`                    | **string**<br/>A single note that can be recorded against an EPC record |
```

| `schemeId`                   | **integer**<br/>The ID of the related scheme                         |
| `scope`                      | **enumeration**<br/>Indicates if the EPC is related to a unit or whole building. Must be a valid item from the [EPC scope](#epc-scope) enumeration                               |
| `unitId`                     | **integer**<br/>The ID of the related unit. This is mandatory when scope is set to "Unit"     |

> | `note`                    | **string**<br/>The latest note that has been recorded on the EPC record |

Reviewer: ```suggestion
| `note`                    | **string**<br/>A single note that can be recorded against an EPC record |
```

    "date": "2021-11-07T08:11:23.813Z",
    "schemeId": 2,
    "scope": "Unit",
    "unitId": 43
    "unitId": 43,
    "note":null
  }
```

> Response (200)

> **Summary:** Updates an existing EPC scheme by ID

Reviewer: Same, this endpoint is for EPC scheme, but it appears to have been changed to just EPC?

**Summary:** Updates an existing EPC by ID

### HTTP Request 

> `PUT /api/v1/epcs/schemes`

Reviewer: Shouldn't this be the /schemes?

| `schemeId`                   | **integer**<br/>The ID of the related scheme                         |
| `scope`                      | **enumeration**<br/>Indicates if the EPC is related to a unit or whole building. Must be a valid item from the [EPC scope](#epc-scope) enumeration                               |
| `unitId`                     | **integer**<br/>The ID of the related unit. This is mandatory when scope is set to "Unit"     |

> | `note`                    | **string**<br/>The latest note that has been recorded on the EPC record |

Reviewer: And so this isn't on the epcScheme?

      "fri": false,
      "epcExempt": false
    },
    "comment": "[2021-11-09] I'm a comment!",

>     "note" : "[2021-11-09] I'm a note!"

Reviewer: ```suggestion
    "note" : "A single note that can be recorded against an EPC record"
```

| `unit.fri`                   | **boolean**<br/>A boolean flag indicating if the current lease is [Full Repairing and Insuring (FRI)](https://www.herrington-carmichael.com/full-repairing-and-insuring-lease/) or not |
| `unit.epcExempt`             | **boolean**<br/>A boolean flag indicating if this unit has an [EPC exemption](https://www.gov.uk/energy-performance-certificate-commercial-property/exemptions) or not |
| `comment`                    | **string**<br/>The latest comment that has been recorded on the EPC record |

> | `note`                    | **string**<br/>The latest note that has been recorded on the EPC record |

Reviewer: ```suggestion
| `note`                    | **string**<br/>A single note that can be recorded against an EPC record|
```
